demo/src/main/java/com/example/DemoApplication.java

```diff
@@ -32,6 +32,9 @@ public void run(String... arg0) throws Exception {
 
 				userRepository.findAll().stream().forEach(user -> System.out.println(user));
 
+				// Intentional bug
+				String str = "";
+
 				System.out.println(userRepository.findAllByName("Raj"));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS106)